.github/workflows/test.yml

```diff
@@ -224,7 +236,12 @@ jobs:
           name: "${{ github.sha }}-aa"
         if: "env.GIT_DIFF != ''"
       - name: Run tests with race detector
-        run: cat xaa.txt | xargs go test -mod=readonly -timeout 15m -race -tags='cgo ledger test_ledger_mock'
+        run: cat xaa.txt | xargs go test -mod=readonly -json -timeout 15m -race -tags='cgo ledger test_ledger_mock' > xaa-race-output.txt
+        if: "env.GIT_DIFF != ''"
```

Comment:
I would go with just `if: env.GIT_DIFF`, to look cleaner. But this is just a suggestion.

Reply:
@marbar3778 thoughts?